vignettes/reuse.Rmd

```diff
@@ -110,27 +110,104 @@ add <- function(x, y) x + y
 times <- function(x, y) x * y
 ```
 
-## Inheriting documentation
+## Automatically copy tags
 
-You can inherit documentation from other topics in a few ways:
+If two or more functions share have similarities but are different or complex enough that you don't want to document them in a single file, you can use one of the four `@inherits` tags to automatically copy various components from another topic:
 
--   `@inheritParams source_function` inherits just the parameter documentation from `source_function()`.
+-   `@inheritParams foo` will copy `@param` contents from `foo`.
+-   `@inherit foo` will copy all supported components from `foo`.
+-   `@inheritSection foo {Section title}` will copy the `@section {Section title}` section from `foo`..
+-   `@inheritDotParams foo` will generate documentation for `…` by coping the documentation for `foo()`'s arguments.
 
--   `@inherit source_function` will inherit all supported components from `source_function`.
+We think of this an "inheritance" rather than just copying, because anything you inherit can be overridden by a more specific definition in the documentation. This applies particularly to `@inheritParams` which allows you to copy the documentation for some parameters while documenting others directly. We'll focus on this first.
```

Comment:
```suggestion
We think of this as "inheritance" rather than just copying, because anything you inherit can be overridden by a more specific definition in the documentation. This applies particularly to `@inheritParams` which allows you to copy the documentation for some parameters while documenting others directly. We'll focus on this first.
```